jenkins-integration/beaker/install/shared/23_install_system_gems.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+test_name 'Install System Gems'
+
+def install_system_gem(host, gem_name)
+  step "Install '#{gem_name}' gem"
+  # Ensure it's the system ruby
+  on(host, "/usr/bin/env gem install #{gem_name}")
+end
+
+step 'Install gems using system ruby'
+
+gem_list = ENV['PUPPET_GATLING_SYSTEM_GEMS'].split(',')
```

Comment:
@jpinsonault you could just hard-code them into the `pipeline.groovy` for now?  like, in that `install_system_gems` method?  I can't think of a reason why it'd be bad to have the hocon gem installed for all the runs for now.